pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -29,7 +29,10 @@ import (
 )
 
 // podStatusManagerStateFile is the file name where status manager stores its state
-const podStatusManagerStateFile = "pod_status_manager_state"
+const (
+	allocatedPodsStateFile = "allocated_pods_state"
+	actuatedPodsStateFile  = "actuated_pods_state"
```

Comment:
Will this need a docs change to https://kubernetes.io/docs/reference/node/kubelet-files/?

(related: should we document this detail even if not already documented?)

Reply:
Thanks for pointing that out, I didn't know these were documented. I added a line item to https://github.com/kubernetes/website/issues/50022